apps/docs/components/.vuepress/components/TypographyList.vue

```diff
@@ -21,15 +21,15 @@
 /* Vuepress can't use the most recent Tailwind version, this cannot be pulled from the plugin,
   hardcoding the values for now. */
 const typographyClasses = [
-  ['display-1', 'fontSize.6xl', 'lineHeight.extra-tight', 'fontFamily.headings'],
-  ['display-2', 'fontSize.4xl', 'lineHeight.10', 'fontFamily.headings'],
-  ['display-3', 'fontSize.2xl', 'lineHeight.8', 'fontFamily.headings'],
-  ['headline-1', 'fontSize.6xl', 'lineHeight.extra-tight', 'fontFamily.headings'],
-  ['headline-2', 'fontSize.4xl', 'lineHeight.10', 'fontFamily.headings'],
-  ['headline-3', 'fontSize.2xl', 'lineHeight.8', 'fontFamily.headings'],
-  ['headline-4', 'fontSize.lg', 'lineHeight.7', 'fontFamily.headings'],
-  ['headline-5', 'fontSize.base', 'lineHeight.6', 'fontFamily.headings'],
-  ['headline-6', 'fontSize.sm', 'lineHeight.5', 'fontFamily.headings'],
+  ['display-1', 'fontSize.6xl', 'lineHeight.extra-tight'],
+  ['display-2', 'fontSize.4xl', 'lineHeight.10'],
+  ['display-3', 'fontSize.2xl', 'lineHeight.8'],
+  ['headline-1', 'fontSize.6xl', 'lineHeight.extra-tight'],
+  ['headline-2', 'fontSize.4xl', 'lineHeight.10'],
+  ['headline-3', 'fontSize.2xl', 'lineHeight.8'],
+  ['headline-4', 'fontSize.lg', 'lineHeight.7'],
+  ['headline-5', 'fontSize.base', 'lineHeight.6'],
+  ['headline-6', 'fontSize.sm', 'lineHeight.5'],
```

Comment:
Comment: this may be painful if someone wants to change the heading font imo

Reply:
thats actually good point, sure will leave it, thanks